Aglis, Entrance
Meru
Retail: Checks if Meru is already at the target location. Poor math precision means he never arrives to the exact
location, so it never exits the subroutine early.
Severed Chains: High precision meets early exit conditions. Early exit doesn't reset to an idle animation so she
sometimes runs in place.
Fix: Bypass unnecessary early exit.
--- original
+++ modified
@@ -149,6 +149,7 @@
 call 97, 0x2
 LABEL_23:
 call 102, stor[0], stor[29], stor[30], stor[31]
+jmp inl[:LABEL_24]
 jmp_cmp !=, stor[29], stor[24], inl[:LABEL_24]
 jmp_cmp !=, stor[30], stor[25], inl[:LABEL_24]
 jmp_cmp !=, stor[31], stor[26], inl[:LABEL_24]
